Backend-Challenges.md

```diff
@@ -87,15 +93,37 @@ When you deploy a smart contract, you will pass these arguments to the construct
 
 ### TokenURI
 
-If you are familiar with [Opensea](https://opensea.io/), you might be wondering: how on earth do they get the NFT images and metadata? Well, often times these images are stored on chain, on IPFS, or even in the cloud. As on-chain storage is expensive, images are more often than not stored off-chain.
+As on-chain storage is expensive, images are more often than not stored off-chain.
+
+If you are familiar with [Opensea](https://opensea.io/), you might be wondering: how do they get the NFT images and metadata?
 
 Your smart contract provides a way to tell Opensea which image belongs to which NFT and where the files are stored through a function called `tokenURI()`. Opensea simply looks for this function in your smart contract and parses the return value.
 
+## Challenge 7 - Minting
+
+The goal of this challenge is to create a minting function in `Challenge_7.sol`, so that the tests in `Challenge_7.t.sol` pass. The mint function should:
+
+- Accept a uint parameter for the mint amount
+- Increase the `totalSupply` by the mint amount
+- Mint the NFTs to `msg.sender` by using ERC721's `_mint()` function.
+
+Some clarification on the new things you will encounter in the test file:
```

Comment:
I wonder, why would you include these clarifications here, and not directly in the code?

Reply:
Good point to clarify though :)

Reply:
It simply "felt" better to me at the time since the explanations are quite verbose to include in the code but can move to the code anyways 👍